openc3/Rakefile

```diff
@@ -84,7 +84,7 @@ task :build => [:devkit] do
   end
 end
 
-task :gems do
+task :gems => [:build] do
```

Comment:
Including the binary files in the gem makes it a binary gem tied to a particular architecture.  So we can't do this...
If you want to be able to point to /openc3 in the Dockerfiles then you need to add rake build to the /openc3 folder as part of the build, not add compiling as a requirement for building the gem.

Reply:
Ok that makes sense. I went ahead and removed this.